Composer/packages/lib/code-editor/demo/src/inlineEditor.tsx

```diff
@@ -34,5 +36,5 @@ export default function App() {
       path: '/lg-language-server',
     },
   };
-  return <LgEditor {...props} />;
+  return <LgCodeEditor {...props} telemetryClient={mockTelemetryClient} />;
```

Comment:
Is using a mock intentional in a non-test component?

Reply:
this is demo code